### PR TITLE
Add additional icon to the dxDropDownButton when splitButton is false

### PR DESCRIPTION
**Pull request by mrjohn0011**

### CODE REVIEW

File: js/ui/drop_down_button.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Popup from "./popup";
 import List from "./list";
 import { compileGetter } from "../core/utils/data";
 import domUtils from "../core/utils/dom";
+import iconUtils from "../core/utils/icon";
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]:
```suggestion
import { getImageContainer } from "../core/utils/icon";
```

File: js/ui/drop_down_button.js [hunk 2]
```diff
@@ -464,6 +463,13 @@ let DropDownButton = Widget.inherit({
         this._bindInnerWidgetOptions(this._popup, "dropDownOptions");
     },
 
+    _renderAdditionalIcon() {
+        const $firstButtonContent = this._buttonGroup.$element().find(".dx-button-content").eq(0);
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]: Extract class name to a const

File: js/ui/drop_down_button.js [hunk 3]
```diff
@@ -464,6 +463,13 @@ let DropDownButton = Widget.inherit({
...
+        const $iconElement = iconUtils.getImageContainer("spindown");
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]:
```suggestion
        const $iconElement = getImageContainer("spindown");
```

File: js/ui/drop_down_button.js [hunk 4]
```diff
@@ -464,6 +463,13 @@ let DropDownButton = Widget.inherit({
...
+        $iconElement.addClass("dx-icon-right");
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]: Extract class name to a const

Comment (DokaRus) [inline js/ui/drop_down_button.js]:
```suggestion
        $iconElement
          .addClass("dx-icon-right")
          .appendTo($firstButtonContent);
```

File: js/ui/drop_down_button.js [hunk 5]
```diff
@@ -477,6 +483,8 @@ let DropDownButton = Widget.inherit({
         this._buttonGroup.registerKeyHandler("upArrow", this._upDownKeyHandler.bind(this));
         this._buttonGroup.registerKeyHandler("escape", this._escHandler.bind(this));
 
+        this.option("splitButton") || this._renderAdditionalIcon();
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]: Make it more clear
```suggestion
        if(!this.option("splitButton")) {
          this._renderAdditionalIcon();
        }
```

File: js/ui/drop_down_button.js [hunk 6]
```diff
@@ -597,11 +605,13 @@ let DropDownButton = Widget.inherit({
                 this._buttonGroup.option("items[0]", extend({}, this._actionButtonConfig(), {
                     icon: value
                 }));
+                this.option("splitButton") || this._renderAdditionalIcon();
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]: extract method?

File: js/ui/drop_down_button.js [hunk 7]
```diff
@@ -597,11 +605,13 @@ let DropDownButton = Widget.inherit({
...
                 break;
             case "text":
                 this._buttonGroup.option("items[0]", extend({}, this._actionButtonConfig(), {
                     text: value
                 }));
+                this.option("splitButton") || this._renderAdditionalIcon();
```
Comment (DokaRus) [inline js/ui/drop_down_button.js]: And there

**Review by DokaRus — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — DISMISSED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**